### ISSUE TITLE
Feature Request: Passthrough single Keys instead of the whole Keyboard

### ISSUE THREAD

**Issue by KGT1** [lal12/macroMyKBD#5]
For example, being able to passthrough the whole numpad but not the rest of the Keyboard.
Current workaround is using AHK, which in my opinion isn't ideal.

**Comment by lal12** [comment]
Yeah probably is a feature worth implementing. However I guess passthrough should be correctly working before (e.g. repeating input while key is hold down).
